RussianFilter/sections/specific.txt

```diff
@@ -90,6 +90,9 @@ rb.ru##.read_also
 ||wrzgu.com^$domain=lenkino.net
 hardwareluxx.ru###rec1
 hardwareluxx.ru###rec2
+||flashplayer.ru/stopvideo*.php
+flashplayer.ru###quiz
+flashplayer.ru##.gametiles > div[id]
```

Comment:
When you hide this element, you should modify the three elements with "style="width: 181px; margin: 0px 0px 379px;"" too, else user has white space.

Reply:
Leftover
![image](https://user-images.githubusercontent.com/8361299/61472112-7847d580-a98c-11e9-96a9-058aedbddf22.png)
But i am not sure if it is ad. Better block in Annoyances.

Reply:
And ad leftover:
`flashplayer.ru##.main > div[id][style*="min-height"][style*="position"][style*="overflow"]`